### PR TITLE
Show user's photo

### PR DESCRIPTION
**Pull request by brunomichetti**
- Show user's photo. If the user doesn't have any photo, then a default image is shown.
- Fix in shapes errors.

Screen Shot 2020-07-29 at 17 44 17
Screen Shot 2020-07-29 at 17 44 30
Screen Shot 2020-07-29 at 17 45 34

### CODE REVIEW

File: src/components/users/UserImageName.js [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
+          width: 150,
+          height: 150,
+          borderRadius: 100,
+          overflow: 'hidden',
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <img src={showImage} width="250" alt="user" />
```
Comment (peter7z) [inline src/components/users/UserImageName.js]:
```suggestion
        <img src={user.profile_picture || defaultProfileImg} width="250" alt="user" />
```

I also think there is no need to set the width. Have you tried?

File: src/components/users/UserImageName.js [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
-        />
+      <div
+        style={{
+          width: 150,
+          height: 150,
+          borderRadius: 100,
+          overflow: 'hidden',
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
```
Comment (peter7z) [inline src/components/users/UserImageName.js]: Why don't you give this div a classname a move these styles to a `css` file?

File: src/components/users/UserImageName.js [hunk 3]
```diff
@@ -1,20 +1,29 @@
 import React from 'react';
-import ExampleComponent from 'react-rounded-image';
 
 import defaultProfileImg from '../../assets/profilePicture.png';
 import './user-profile.scss';
 import { userShape } from '../../constants/shapes';
 
 const UserImageName = ({ user }) => {
```
Comment (tamarasuarezrod) [inline src/components/users/UserImageName.js]:
```suggestion
const UserImageName = ({ user: { profile_picture } }) => {
```

**Review by tamarasuarezrod — APPROVED** [review]

File: src/components/users/UserImageName.js (hunk 2 above)
Comment (brunomichetti) [reply src/components/users/UserImageName.js]: You're right! My bad

**Review by manema — APPROVED** [review]

**Review by peter7z — APPROVED** [review]

**Pull request merged**